Python/ogusa/SS.py

```diff
@@ -516,7 +516,7 @@ def function_to_minimize(chi_params_scalars, chi_params_init, tax_params, params
     wguess, rguess, factorguess, T_Hguess = solutions[(2 * S * J):]
     solutions = SS_solver(b_guess.reshape(S, J), n_guess.reshape(S, J), wguess,
                           rguess, T_Hguess, factorguess, chi_params_init[J:],
-                          chi_params_init[:J], params, iterative_params,
+                          chi_params_init[:J], tax_params, params, iterative_params,
```

Comment:
The `tax_params` argument seems to be a necessary argument of the function, so I don't know how this was working before.